express-get-json/server.js

```diff
@@ -0,0 +1,37 @@
+import express from 'express';
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95,
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100,
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92,
+  },
+};
+const app = express();
+
+app.listen(8080, () => {
```

Comment:
By convention, we usually put this at the bottom of the file so the code reads as "set up the endpoints, then listen".